client/app/bundles/course/assessment/pages/AssessmentStatistics/LiveHelpStatisticsTable.tsx

```diff
@@ -94,7 +94,8 @@ const LiveHelpStatisticsTable: FC<Props> = (props) => {
   const [parsedStatistics, setParsedStatistics] = useState<
     AssessmentLiveFeedbackStatistics[]
   >([]);
-  const [maxFeedbackCount, setMaxFeedbackCount] = useState<number>(0);
+  const [upperQuartileFeedbackCount, setUpperQuartileFeedbackCount] =
```

Comment:
this is not possible because the implementation is tangled up together with the implementation of live feedback history

Reply:
how can the implementation be tangled?
I don't mean to squash the 2 commits, since the commit that change `MaxFeedback` to `UpperQuartileFeedback` is indeed done after you implemented the LiveFeedbackHistory

What I mean is you need to
- rebase the commits, set `edit` to the commit https://github.com/Coursemology/coursemology2/commit/7ab29dd53db5745538f5f89afda3c1ce82716a19
- Then you edit all the `maxFeedback` related to `upperQuartileFeedback`

There might be some merge conflict though, in that case, it's your task also to resolve all the conflicts. This might be quite tedious to do but I assure you this is the right thing to do.

Reply:
done